addons/sourcemod/scripting/scp_sf/defines.sp

```diff
@@ -21,6 +21,10 @@
 #define ITEM_INDEX_SCP18 30018
 #define ITEM_INDEX_DISARMER 954
 
+#define DEFINDEX_UNDEFINED 65535
```

Comment:
A proper solution would be to have `DEFINDEX_UNDEFINED` as `-1` and getting def index as `uint16` from `defindex & 0xFFFF`, whenever if that's worth putting extra work to it.

Reply:
If the defindex is `65535`, then `65535 & 0xFFFF` is just going to spit the same number out again. I think this is fine as-is.